docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -121,16 +105,17 @@ And that's it!  Although this compiled only for .NET Framework 4.0, you can use
 
 **NOTE:** These instructions assume you have .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
 
-Targeting a PCL profile is a bit trickier.  For starters, [reference this list of PCL profiles](http://embed.plnkr.co/03ck2dCtnJogBKHJ9EjY/preview) to make sure you have the correct target.  Hover over the name of each entry for the framework identifier, which you will need.
+Targeting a PCL profile is a bit trickier than targeting .NET Standard or .NET Framework.  For starters, [reference this list of PCL profiles](http://embed.plnkr.co/03ck2dCtnJogBKHJ9EjY/preview) to find the NuGet target which corresponds to the PCL profile you are targeting.
 
-You will then need to do two things:
+You will then need to do the following:
 
-1. List the dependencies for each target *inside* of that target's node in the `project.json`.  The "global" `dependencies` section cannot be used because these dependencies are framework-specific.
-2. List the framework assemblies you are using in your code *inside* the `frameworkAssemblies` node.  This will require, at a minimum, `mscorlib`, `System`, and `System.Core`.  For other APIs (such as `System.Linq`), you will need to find the assembly which corresponds to the API call you're using.  You'll need to then manually list that assembly.
+1. Create a new entry under `frameworks` in your `project.json`, named `.NETPortable,Version=v{version},Profile=Profile{profile}`, where `{version}` and `{profile}` correspond to a PCL version number and Profile number, respectively.
+2. In this new entry, list every single assembly used for that target under a `frameworkAssemblies` entry.  This includes `mscorlib`, `System`, and `System.Core`.
+3. If you are multitargeting (see the next section), you'll have to explicitly list dependencies for each target under their target entries.  You won't be able to use a global `dependnecies` entry anymore.
```

Comment:
I think you can use both at the same time, not sure...

If you are multitargeting, target specific dependencies need to be placed under their target entry. Dependencies common to all targets can be placed in the global dependencies.